tests/jerry/es.next/module-dynamic-import.js

```diff
@@ -0,0 +1,28 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
```

Comment:
We only have a test which is expected to fail? Could we have a single js test where the "correct" execution is tested?

Reply:
I mentioned above that no default implementation is present for modules at the moment, because the user value has no fixed format. So currently it always fails.